app/src/main/java/org/greenstand/android/TreeTracker/models/messages/Converters.kt

```diff
@@ -5,21 +5,24 @@ import org.greenstand.android.TreeTracker.models.messages.network.responses.Ques
 
 fun QuestionResponse.toQuestion(): Question {
     return Question(
-        prompt = prompt!!,
-        choices = choices!!
+        prompt = prompt ?: "NULL PROMPT",
+        choices = choices ?: emptyList()
     )
 }
 
 fun MessageResponse.toMessage(): Message {
     return when {
-        survey.questions != null -> return SurveyMessage(
-            from = from,
-            to = to,
-            composedAt = composedAt,
-            questions = survey.questions.map { it.toQuestion() },
-            answers = survey.answers!!
-        )
+        survey.questions.isNotEmpty() && survey.questions.first().prompt != null ->
```

Comment:
survey.questions.first().prompt could be "NULL PROMPT" as per changes in line 8 of the Converters.kt file and wrongfully satisfy this condition since it is not null.